cfile/cfile.cpp

```diff
@@ -103,42 +94,38 @@ static CFILE *open_file_in_lib(const char *filename);
 // NOTE:	libname must be valid for the entire execution of the program.  Therefore, it should either
 //			be a fully-specified path name, or the current directory must not change.
 // Returns: 0 if error, else library handle that can be used to close the library
-int cf_OpenLibrary(const char *libname) {
+int cf_OpenLibrary(const std::filesystem::path &libname) {
   FILE *fp;
   int i;
   uint32_t offset;
   static int first_time = 1;
-  tHogHeader header;
-  tHogFileEntry entry;
+  tHogHeader header{};
+  tHogFileEntry entry{};
 
   // allocation library structure
   std::shared_ptr<library> lib = std::make_shared<library>();
 #ifdef __LINUX__
```

Comment:
Didnt you say Mac is also case insensitive? OSX defines `__LINUX__`

Reply:
This gray area that was discovered after unittests implementation. Still, there nothing bad happens on macos, effectively that logic works. Removed here ifdef